app/questionnaire/navigator.py

```diff
@@ -104,54 +19,74 @@ def __init__(self, survey_json, metadata=None, answer_store=None):
         self.first_block_id = SchemaHelper.get_first_block_id(self.survey_json)
         self.first_group_id = SchemaHelper.get_first_group_id(self.survey_json)
         self.last_group_id = SchemaHelper.get_last_group_id(self.survey_json)
+
+        self.first_block_location = {
+            "group_id": self.first_group_id,
+            "group_instance": 0,
+            "block_id": self.first_block_id,
+        }
         self.location_path = self.get_location_path()
+        self.first_location = self.location_path[0]
```

Comment:
Nope, not always. First_location points to everything, including interstital. Block location only includes blocks.